src/test/resources/org/jenkinsci/plugins/stashNotifier/ConfigAsCodeTest/configuration-as-code.yml

```diff
@@ -0,0 +1,10 @@
+unclassified:
+  notifyBitbucket:
+    considerUnstableAsSuccess: true
+    credentialsId: "bitbucket-credentials"
+    disableInprogressNotification: true
+    ignoreUnverifiedSsl: true
+    includeBuildNumberInKey: true
+    prependParentProjectKey: true
+    projectKey: "JEN"
+    stashRootUrl: "https://bitbucket.com"
```

Comment:
Even if it's only an example value, I suggest to use something like `https://my.company.intranet/bitbucket`. This more clearly states that this plugin is about "Bitbucket Server" and not "Bitbucket Cloud". Also see #225 for a cleanup regarding this.